src/main/resources/config/liquibase/changelog/20211216000000_changelog.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <property name="autoIncrement" value="true"/>
+
```

Comment:
Could you add some comments on what you change here so it's easier to read as it's quite a huge change?
Double-check locally though, I don't know how the checksum gets generated and if that would cause issues with the already deployed changes

Reply:
Tested it locally and when it is outside of the changeset tag it works fine.